### DISCUSSION TITLE
`no:tracks` and `-no:tracks` filter on GitHub Projects doesn't work properly

### DISCUSSION THREAD

**Discussion by 8ma10s** [community/community#46242] [Projects and Issues]
### Select Topic Area

Bug

### Body

When we filter by `no:tracks`, we expect to see every issue that has no GitHub tasklists == issues that is not tracking any issue.

However, for some reason, even the issues that is not tracking any other issues are gone from the view once I enable `no:tracks` filter.

Also, when we filter by `-no:tracks`, we expect to see every issue that's tracking at least 1 other issue.
However, I see that some issues that's not tracking any issue is still remaining in the filtered view even after I enable the filter.

## Examples

This is my table view. 
![スクリーンショット 2023-02-03 15 40 32](https://user-images.githubusercontent.com/24708144/216530829-a753d4a0-bafe-400d-a929-17b19b58f334.png)

When i filter by `no:tracks`, I would expect to see 409, 416, 417... since they're not tracking any issues.
However, this is what I actually get.
![スクリーンショット 2023-02-03 15 41 04](https://user-images.githubusercontent.com/24708144/216531081-7c498744-d040-4f24-93a3-b0e66bce605a.png)

When I filter by `-no:tracks`, I would expect to see 420,426,428 since they're tracking something.
However, this is what I actually get.

![スクリーンショット 2023-02-03 15 40 47](https://user-images.githubusercontent.com/24708144/216531199-b8d300ce-3dee-4517-9042-3bc04ad3bede.png)

I think the `tracks`  filter is not working correctly.

**Comment by rileybroughten** [comment]
Thank you @ssym0614! I raised this with the team and there has been a bug filed for it, will keep you posted with updates!
